tests/components/roku/test_media_player.py

```diff
@@ -74,6 +78,7 @@ async def test_setup(
 
     assert hass.states.get(MAIN_ENTITY_ID)
     assert main
+    assert main.device_class == DEVICE_CLASS_RECEIVER
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6103615)